code/espurna/sensor.ino

```diff
@@ -462,18 +462,37 @@ void _sensorResetTS() {
     #endif
 }
 
-double _sensorEnergyTotal() {
+double _sensorEnergyTotal(unsigned int index = -1) {
```

Comment:
I missed this earlier... note the `unsigned` :) Fixed via #1875

Reply:
Thank you!